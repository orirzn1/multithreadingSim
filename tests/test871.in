L12
S5
O9
N5

T0
I@0x00000000
LOAD $0, $3, $6
LOAD $6, $1, $2
STORE $2, $5, $1
ADD $1, $0, $7
STORE $0, $6, $0
ADD $0, $6, $5
ADDI $4, $1, 0x4
SUBI $3, $6, 0x1
LOAD $4, $4, $6
LOAD $4, $2, 0x1
HALT $0

T1
I@0x00000000
SUB $5, $3, $0
ADDI $6, $6, 0x3
SUBI $2, $5, 0x1
SUBI $4, $2, 0x1
STORE $1, $3, $3
STORE $4, $7, $7
ADD $0, $2, $0
STORE $5, $5, $3
HALT $0

T2
I@0x00000000
LOAD $5, $4, $7
SUB $5, $7, $4
LOAD $1, $0, 0x3
STORE $7, $2, $0
SUB $3, $3, $5
ADDI $5, $4, 0x1
SUB $7, $1, $4
ADDI $4, $6, 0x4
HALT $0

T3
I@0x00000000
SUBI $7, $0, 0x1
SUB $7, $6, $6
ADD $4, $4, $3
ADDI $5, $2, 0x2
SUBI $2, $2, 0x3
LOAD $1, $1, 0x2
STORE $1, $7, $4
SUBI $1, $1, 0x4
ADD $2, $7, $2
STORE $4, $4, 0x2
HALT $0

T4
I@0x00000000
STORE $1, $1, 0x2
SUBI $6, $2, 0x4
SUBI $5, $3, 0x4
SUBI $5, $6, 0x3
SUB $3, $4, $7
SUBI $6, $3, 0x4
SUBI $1, $7, 0x4
SUB $5, $1, $5
SUB $6, $6, $5
STORE $7, $6, $7
HALT $0

D@0x0
0xA
0x8
0x2
0x8
0x3
0x5
0x4
0x8
0xA
