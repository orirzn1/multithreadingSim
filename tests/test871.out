
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0xA	R2 = 0x0	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x3	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x1	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x2	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x3	R6 = 0xFFFFFFF9	R7 = 0x0
Blocked MT CPI for this program 4.823529

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x3	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x1	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x2	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x3	R6 = 0xFFFFFFF9	R7 = 0x0
Finegrained Multithreading CPI for this program 1.509804

